ml/Neural Network/Requirements.txt

```diff
@@ -0,0 +1,9 @@
+NOTE : I am not sure about lower version but i have this version in my pc 
```

Comment:
Hey @manthan89-py, we don't write any NOTES in the requirements.txt. The version specified along with the libraries already does that for you. Please remove this as well as the bottom-most comment.